src/forms/SubmissionDetail.jsx

```diff
@@ -42,6 +42,11 @@ export default class SubmissionDetail extends Component {
       <div style={styles.answersContainer}>
         <div style={styles.answers}>
           {submission.replies.map((reply, key) => {
+            
+            // To filter null dates from the replies.
+            if (reply.answer.value === '--') {
+              return null;
```

Comment:
@impronunciable The reason why I didn't use that is because it's also looping through all the anwser values. (I couldn't find a type field in the answer object) for doing that I need to make sure that .value only comes from dates. Otherwise nothing will show up 
<img width="427" alt="screen shot 2016-09-14 at 1 35 23 pm" src="https://cloud.githubusercontent.com/assets/1401559/18520889/2064eb8e-7a80-11e6-96f5-5751e660c065.png">

@impronunciable @riley